src/scene/ScreenPosition.py

```
@@ -0,0 +1,69 @@
+from ok import Box
+
+
+class ScreenPosition:
+    """
+    根据屏幕宽高生成各个位置的 Box。
+    支持：
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)